### DISCUSSION TITLE
Allow Adding more Languages

### DISCUSSION THREAD

**Discussion by engahmadkadi** [ENDER-9/Vocascan#105] [Ideas]
Hi Vocascan,

Is it possible to allow the user to add more elements to the languages' list? I'm thinking of using the platform to learn how to avoid the common mistakes in a language. So for example, one should be able to add a language with name `broken English`. This will be the first side of the ticket to show, and the back side should contain the proper sentence or word in English.

Thanks in advance!

**Comment by luwol03** [comment]
Hey engahmadkadi,

you can already create multiple packages with the same source and destination language with different names and descriptions.

However adding custom languages on user level is planed and the code is already prepared to simply add that feature (all available options are currently saved in the database).

You can also create an issue for that if you want.

I personally also think of using vocascan not only for language learning but also for learning in general for every kind of thing (e.g. physics exam).

luwol03

**Comment by engahmadkadi** [comment]
Hi luwol03,

> you can already create multiple packages with the same source and destination language with different names and descriptions.

I have already used this method to test learning how to distinguish and correct proper sentences from a grammatically wrongs ones in the same language. However, it looks weird and confusing once one starts learning from this packages, as both flags and language names are the same.

> I personally also think of using vocascan not only for language learning but also for learning in general for every kind of thing (e.g. physics exam).

Your sentence really inspired me. You're right, this could be apply for everything to learn. Probably on the future also for card games, where you have questions at one side, and answers at the back!

Ahmad
